Dockerfile

```diff
@@ -66,38 +70,28 @@ RUN echo "---- INSTALL RUNTIME PACKAGES ----" && \
   php7-phar \
   php7-tokenizer \
   php7-xmlwriter \
-  php7-openssl \
-  && echo "date.timezone = UTC" >> /etc/php7/php.ini
-
-
+  php7-openssl && \
+  echo "date.timezone = UTC" >> /etc/php7/php.ini
 
 # copy ffmpeg static with libfdk from mwader docker image
-COPY --from=mwader/static-ffmpeg:4.2.2 /ffmpeg /usr/local/bin/
+COPY --from=mwader/static-ffmpeg:4.3 /ffmpeg /usr/local/bin/
 
 # copy compiled mp4v2 binaries, libs and fdkaac encoder to runtime image
 COPY --from=builder /usr/local/bin/mp4* /usr/local/bin/fdkaac /usr/local/bin/
 COPY --from=builder /usr/local/lib/libmp4v2* /usr/local/lib/
 
-
 ARG M4B_TOOL_DOWNLOAD_LINK="https://github.com/sandreas/m4b-tool/releases/latest/download/m4b-tool.tar.gz"
 
 # workaround to copy a local m4b-tool.phar IF it exists
 ADD ./Dockerfile ./dist/m4b-tool.phar* /tmp/
-RUN echo "---- INSTALL M4B-TOOL ----" \
-    && if [ ! -f /tmp/m4b-tool.phar ]; then \
-            wget "${M4B_TOOL_DOWNLOAD_LINK}" -O /tmp/m4b-tool.tar.gz && \
-            if [ ! -f /tmp/m4b-tool.phar ]; then \
-                tar xzf /tmp/m4b-tool.tar.gz -C /tmp/ && rm /tmp/m4b-tool.tar.gz ;\
-            fi \
-       fi \
-    && mv /tmp/m4b-tool.phar /usr/local/bin/m4b-tool \
-    && M4B_TOOL_PRE_RELEASE_LINK=$(wget -q -O - https://github.com/sandreas/m4b-tool/releases/tag/latest | grep M4B_TOOL_DOWNLOAD_LINK | cut -d '=' -f 2 | cut -d ' ' -f 1) \
-    && wget "${M4B_TOOL_PRE_RELEASE_LINK}" -O /tmp/m4b-tool.tar.gz \
-    && tar xzf /tmp/m4b-tool.tar.gz -C /tmp/ && rm /tmp/m4b-tool.tar.gz \
-    && mv /tmp/m4b-tool.phar /usr/local/bin/m4b-tool-pre \
-    && chmod +x /usr/local/bin/m4b-tool /usr/local/bin/m4b-tool-pre
 
-WORKDIR ${WORKDIR}
+RUN echo "---- INSTALL M4B-TOOL ----" && \
```

Comment:
This part is missing some things:
- A custom local build, that is copied into the image, if present (this is important for my personal tests)
- The process will only install the latest pre-release, while it before installed both the latest pre-release as `m4b-tool-pre` AND the stable release as `m4b-tool`

Reply:
I will add these back in then. Sorry for the delay!